HouseRules_Essentials/Rules/PartyElectricityDamageOverriddenRule.cs

```diff
@@ -0,0 +1,104 @@
+﻿namespace HouseRules.Essentials.Rules
+{
+    using Boardgame;
+    using Boardgame.BoardEntities;
+    using Boardgame.BoardEntities.Abilities;
+    using Boardgame.GameplayEffects;
+    using Boardgame.SerializableEvents;
+    using DataKeys;
+    using HarmonyLib;
+    using HouseRules.Types;
+    using UnityEngine;
+    using System.IO;
+
+    public sealed class PartyElectricityDamageOverriddenRule : Rule, IConfigWritable<bool>, IPatchable,
+        IMultiplayerSafe
+    {
+        public override string Description => "Player on player electricity damage is zero.";
+
+        private static bool _globalAdjustments;
+        private static bool _isActivated;
+
+        private readonly bool _adjustments;
+        private static GameContext _gameContext;
+
+        public PartyElectricityDamageOverriddenRule(bool adjustments)
+        {
+            _adjustments = adjustments;
+        }
+
+        public bool GetConfigObject() => _adjustments;
+
+        protected override void OnActivate(GameContext gameContext)
+        {
+            _globalAdjustments = _adjustments;
+            _isActivated = true;
+            _gameContext = gameContext;
+        }
+
+        protected override void OnDeactivate(GameContext gameContext) => _isActivated = false;
+
+        private static void Patch(Harmony harmony)
+        {
+
+            harmony.Patch(
+                original: AccessTools.Method(typeof(ProjectileHitSequence), "OnStarted"),
+                prefix: new HarmonyMethod(
+                    typeof(PartyElectricityDamageOverriddenRule),
+                    nameof(ProjectileHitSequence_OnStarted_Prefix)));
+
+            harmony.Patch(
+                original: AccessTools.Method(typeof(SerializableEventQueue), "ValidateSerializableEvent"),
+                prefix: new HarmonyMethod(
+                    typeof(PartyElectricityDamageOverriddenRule),
+                    nameof(SerializableEventQueue_ValidateSerializableEvent_Postfix)));
```

Comment:
This patch shouldn't have been included at all... It was a WIP commit on the DontShockFriends branch from when I was trying to debug what was going on with the statusEffect sync. KHyperia suggested adding it as a workaround because the serializable event wasn't allowed by default. I've now removed it.